src/App.css

```diff
@@ -0,0 +1 @@
+  
```

Comment:
It's an empty so delete it and delete the related sentence in App.js

Reply:
done :octocat: